src/cap-description.adoc

```diff
@@ -429,15 +427,18 @@ and a carry out is implied if `T[MW - 3:0] < B[MW - 3:0]` since it is
 guaranteed that the top is larger than the base.
 
 The compressed bounds encoding allows the address to roam over a large
-_representable_ region while maintaining the original bounds. This relies on
-using the 'spare' encodings where `T < B` to define a space boundary R,
-relative to the base, calculated by subtracting 1 from the top three bits of B.
+_representable_ region while maintaining the original bounds. This is enabled by
+defining a lower boundary R from the out-of-bounds values that allows us to
+disambiguate the location of the bounds with respect to an out-of-bounds address.
+R is caluculated
+relative to the base by subtracting 2^MW-2^ from B.
 If B, T or _a_[E + MW - 1:E] is less than R, it is inferred that they lie in the
 2^E+MW^ aligned region above R labelled space~U~ in
 xref:cap_bounds_map[xrefstyle=short] and the corrections _c~t~_ and _c~b~_ are
-computed accordingly. The overall effect is that at least 2^E+MW^/8 bytes below
-the base address and 2^E+MW^/4 bytes above the top address can roam
-out-of-bounds while still allowing the bounds to be correctly decoded.
+computed accordingly. The overall effect is that the address can roam
+2^E+MW^/4 bytes below
+the base address and at least 2^E+MW^/4 bytes above the top address
+while still allowing the bounds to be correctly decoded.
 
 [#cap_bounds_map]
 .Memory address bounds encoded within a capability
```

Comment:
Btw this diagram will become out of date with this change.

Reply:
I think the diagram is still correct, this just changes how far the representable space can extended past the bounds in each direction.